omas/machine_mappings/_efit.json

```diff
@@ -291,10 +291,6 @@
   "eval2TDI": "py2tdi(stack_outer,'\\{EFIT_tree}::TOP.MEASUREMENTS.CHIECC','\\{EFIT_tree}::TOP.MEASUREMENTS.CHIFCC')",
   "treename": "{EFIT_tree}"
  },
- "equilibrium.time_slice.:.constraints.pressure.:.position.psi": {
-  "eval2TDI": "py2tdi(efit_psi_to_psi,'\\{EFIT_tree}::TOP.MEASUREMENTS.RPRESS','\\{EFIT_tree}::TOP.RESULTS.GEQDSK.SSIMAG','\\{EFIT_tree}::TOP.RESULTS.GEQDSK.SSIBRY')",
-  "treename": "{EFIT_tree}"
- },
```

Comment:
We should add these as OMAS extensions since they will be in the next IMAS release.  I will work on that

Reply:
Added in following the the IMAS update